exasol_transformers_extension/udfs/models/token_classification_udf.py

```diff
@@ -55,6 +59,7 @@ def execute_prediction(self, model_df: pd.DataFrame) -> List[Union[
         :return: List of dataframe includes prediction details
         """
         text_data = list(model_df['text_data'])
+        #todo  pull relevant part of text data?
```

Comment:
Text data should be the relevant part and not more, the docid, begin and end are more like an id for this text_data.

Reply:
added a comment for now